js/src/logger.ts

```diff
@@ -59,33 +60,36 @@ export interface Span {
   log(event: ExperimentLogPartialArgs): void;
 
   /**
-   * Create a new span. This is useful if you want to log more detailed trace information beyond the scope of a single log event. Data logged over several calls to `Span.log` will be merged into one logical row.
+   * Create a new span and run the provided callback. This is useful if you want to log more detailed trace information beyond the scope of a single log event. Data logged over several calls to `Span.log` will be merged into one logical row.
```

Comment:
I shifted the bulk of the documentation to `traced` rather than `startSpan`, because most of the times people should be calling `traced` in javascript. I also made one breaking change to make the `name` an optional parameter instead of required. This is only breaking in JS, because in python it retains the same position in the arglist, but in JS we move it into the object.

I felt like optional name everywhere was the simplest DX and hopefully we can ask folks to make the slight modification to anywhere they are tracing with spans in JS.

Reply:
So now does `experiment.traced` have the same structure/arguments as `span.traced`?

Reply:
Yes now all of the `traced` methods have the same signature.